src/data-catalog/DataBrowserDetails.test.ts

```diff
@@ -0,0 +1,85 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { getDatasetReleasePoliciesDisplayInformation, useDataCatalog } from 'src/data-catalog/data-browser-utils';
+import { DataBrowserDetails, MainContent, MetadataDetailsComponent } from 'src/data-catalog/DataBrowserDetails';
+import { TEST_DATASET_ONE, TEST_DATASET_TWO, TEST_DATASETS } from 'src/data-catalog/test-datasets';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+
+const mockGoBack = jest.fn();
+
+type NavExports = typeof import('src/libs/nav');
+jest.mock(
+  'src/libs/nav',
+  (): NavExports => ({
+    ...jest.requireActual('src/libs/nav'),
+    history: { goBack: () => mockGoBack() },
```

Comment:
I think you can reference the method directly
```suggestion
    history: { goBack: mockGoBack },
```

Reply:
I actually can't! It has to do with mapping `jest.fn` and the way that it gets picked up by jest (https://stackoverflow.com/questions/65554910/jest-referenceerror-cannot-access-before-initialization)